Bridge Deck Tweaks

CakeQ opened the pull request:
🆑 Cakey
maptweak: adjusted the bridge deck slightly:
maptweak: straightened the outer hallways to be more convenient
maptweak: reorganised the meeting room to use its space better
maptweak: matched wood types accross all offices
maptweak: gave an extra console to the side-islands in the bridge
maptweak: reshuffled COS and CE office contents, added disposal chutes
maptweak: brought outer hallway decals in line with door marking standard
/🆑

https://forums.baystation12.net/threads/bridge-deck-tweaks.7974/

![image](https://user-images.githubusercontent.com/17550641/58366874-4d7f5a80-7ed0-11e9-9c05-f88dd41ad2fc.png)

Primary issues I wanted to fix with the current layout:
- crooked hallways
![image](https://user-images.githubusercontent.com/17550641/58379936-3a3cbf80-7fa2-11e9-999d-473ecaacf505.png)

- airlock funnel (closet misplaced in screenshot)
![image](https://user-images.githubusercontent.com/17550641/58379941-44f75480-7fa2-11e9-91c1-e58fdeff3b15.png)

- cornerdoors
![image](https://user-images.githubusercontent.com/17550641/58379956-6b1cf480-7fa2-11e9-98c6-8dfed092c81e.png)

- office layout improvements
![image](https://user-images.githubusercontent.com/17550641/58379958-7a03a700-7fa2-11e9-9dfa-cf3af46e5d7d.png)

CakeQ committed:
Bridge deck tweaks

minor tweaks to bridge deck

final bridge tweaks

removes rogue wire on bridge

adds missing firedoors to bridge

removes rogue carpets

removes excessive firedoors

Removes faulty manual from bridge

Final bridge tweaks

updates bridge for glass2

Adds missing labeler

Albens commented:
This reverts prior to #25581
This will conflict with #25628 and this doesn't seem to contain any bugfixes.
The stated changelog doesn't accurately reflect the changes.
A more accurate changelog would include

> maptweak: Reverted B-Deck Stairway landing
maptweak: Reverted B-Deck Checkpoint
maptweak: Reverted COS and CE's offices to previous locations, mostly retaining their new layout.  
maptweak: Modified the Meeting Room, by shifting into towards the outer hallways, adding three tables, and shifting the layout.
maptweak: Removes no smoking signs in CE, COS, CSO, XO, SEA Offices and/or Quarters, and Bridge Storage.

The title could also reflect the reverts, as they are the largest functional change. 

> Bridge Deck Reverts, Meeting room changes, minor tweaks

Would be more accurate

This fails to follow the Smoking signs standard, by actively removing them in only specific offices. It's outlined in #25581, but I'll paste it here.

> Adds in NO SMOKING signs to offices which are of average size and don't contain doubled atmospheric scrubbers/pumps, and are more of a badge of position (Ie XO/CMO) and not a primary work space.
The criteria for SMOKING signs are doubled atmospheric scrubbers/pumps, and offices that are a Primary work space (IE SCGR/CL) or special areas (CO Office, Deliberation, Meeting room).

We should attempt to maintain Consistent Mapping Standards, as you yourself have said previously.
I believe you're already aware of the janky piping and wiring at the Bridge Access Hallway(s), and of the bad wires, and of the duplicated ATM's. 
Additionally, you've only partially migrated the XO's office over from mahogany, and have left mahogany tables in. I believe this conflicts with your reasoning about consistent wood types, and is probably an oversight.
You've reintroduced rollerchairs, which where removed from all offices, by giving one only to the XO.

I'll send the constructive or subjective stuff in #feedback. 

Minimized with changes. Will repost if still appropriate based off of technical details when I awake.

CakeQ commented:
Updated PR with changes from #25581 
Also factors in changes from #25628

Replaced all mahogany tables with walnut (oversight)
All travis errors also fixed

SierraKomodo commented:
Some issues I noticed in mapdiff:
You're missing the firedoors I added to the security checkpoint windoor and the shutters in the fore bridge hallway (That separates port from starboard). Not sure about the others yet because mapdiff bot doesn't render firedoors on airlock tiles. The other airlocks that should have firedoors added from my PR:
 - CE's Office.
 - Auxiliary EVA.
 - CO's office bathroom

Bridge deck safe room is missing a fire alarm

CMO's office authenticator and buttons were removed?

I posted more stuff in #feedback that wasn't something explicitly bad, but was still missing from my changes.

CakeQ commented:
@sierrakomodo addressed all concerns

afterthought2 commented:
This has received extremely strong community (and staff) pushback for a minor map tweak PR. Make a thread https://forums.baystation12.net/view/gameplay-changes.104/ here explaining the changes in the next 24 hours. 

The thread will be heavily moderated by me, with an emphasis on civility. Ad hominum attacks on the author or the process will result in repo warnings or worse.

CakeQ commented:
Thread here: https://forums.baystation12.net/threads/bridge-deck-tweaks.7974/

CakeQ commented:
This has been rebased

CakeQ commented:
Had to add a missing hand labeler but this is now finished
